sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultRoleScope.cs

```diff
@@ -1,31 +1,14 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 using System;
-using System.ComponentModel;
 using Azure.Core;
 
-namespace Azure.Security.KeyVault.Administration
+namespace Azure.Security.KeyVault.Administration.Models
 {
-    /// <summary>
-    /// A scope of the role assignment.
-    /// </summary>
-    public readonly struct KeyVaultRoleScope : IEquatable<KeyVaultRoleScope>
+    [CodeGenModel("RoleScope")]
+    public readonly partial struct KeyVaultRoleScope : IEquatable<KeyVaultRoleScope>
```

Comment:
Nit: FWIW you don't need to "re-implement" any interfaces on a partial. The generated one will take care of that.